Fix for yaml front matter on Windows machines.

avishnyak opened the pull request:
Fixes #5 
Fixes formatting

avishnyak committed:
Fix for yaml front matter on Windows machines.

sojournerc commented on test/test.mocha.js:
```diff
@@ -42,7 +42,7 @@ describe('gulp-markdown-docs', function() {
     });
 
-    it('should parse one file', function(done) {
+    it.only('should parse one file', function(done) {
```
remove only

avishnyak replied:
Will do.

sojournerc reviewed (commented):
Thanks for the PR! - I hope I'm not missing anything amongst the formatting changes. 

Do all tests pass after you remove the `.only`?

avishnyak commented:
The tests didn't pass for me even before I started so I thought they needed to be fixed (which is why I forgot to remove the ```.only```).

avishnyak committed:
Fix bug that crept in from using lodash to generate the meta tag.
Fix mocha tests and fixtures.

avishnyak commented:
@sojournerc I compared the fixtures to the data generated by the mocha tests.  

- There were a number of changes related to an updated CSS from highlight.js which I fixed by modifying the fixtures.
- There was 1 bug related to the meta attributes expecting an undefined where it was actually an empty object from the lodash function - I fixed this.
- The tests still don't pass on my machine but manually diffing the results of the mocha test to the fixtures shows 0 diffs.  I suspect this is a line ending situation in the compare since it's not a real diff that knows how to deal with whitespace and line ending differences.

sojournerc commented:
Admittedly the tests are a bit fragile. I've pulled your branch and updated the tests (to work with unix line endings). Everything looks good.

I'm going to bump the major version as this adds the new library which could have unexpected consequences out in the wild. New version is `1.0.0`

Not sure what the best approach is to have test fixtures work with all platforms. I'm certainly open to suggestions.
